internal/engine/wazevo/backend/isa/arm64/lower_instr.go

```diff
@@ -321,6 +332,54 @@ func (m *machine) lowerIcmp(si *ssa.Instruction) {
 	m.insert(cset)
 }
 
+func (m *machine) lowerBand(si *ssa.Instruction) {
+	x, y := si.BinaryData()
+	if !x.Type().IsInt() {
+		panic("BUG?")
+	}
+
+	xDef, yDef := m.compiler.ValueDefinition(x), m.compiler.ValueDefinition(y)
+	rn := m.getOperand_SR_NR(xDef, extModeNone)
```

Comment:
I think only rm register can be "Shifted Register" operand
```suggestion
	rn := m.getOperand_NR(xDef, extModeNone)
```

Reply:
e.g.  https://developer.arm.com/documentation/ddi0596/2020-12/Base-Instructions/EOR--shifted-register---Bitwise-Exclusive-OR--shifted-register--?lang=en

> Bitwise Exclusive OR (shifted register) performs a bitwise Exclusive OR of a register value and an optionally-shifted register value, and writes the result to the destination register.